### PR TITLE
fix: F11 fullscreen and toggle

### PR DESCRIPTION
**Pull request by ericawright**
@bwinton my VMs are acting up and I can't test this at all - I believe this should fix the first of the two F11 issues. Will check once I'm on my other computer, but if you'd like to check on windows that would be great too. 

https://github.com/bwinton/TabCenter/issues/951
https://github.com/bwinton/TabCenter/issues/952

### CODE REVIEW

**Comment by bwinton** [comment]
Doesn't seem to fix #951 or #952 on Windows 10.  😢

**Comment by ericawright** [comment]
This should be ready now

File: skin/base.css [hunk 1]
```diff
@@ -169,8 +169,10 @@
   transition: none;
 }
 
-#main-window[F11-fullscreen] #verticaltabs-box:hover {
+#main-window[F11-fullscreen] #verticaltabs-box:hover,
+#main-window[F11-fullscreen] #verticaltabs-box[tour="true"] {
   width: var(--pinned-width) !important;
+  /*can cause problems - pinned width*/
```
Comment (bwinton) [inline skin/base.css]: How can this cause problems?

File: main.js [hunk 2]
```diff
@@ -468,6 +609,11 @@ exports.onUnload = function (reason) {
       }
 
       win.VerticalTabs.unload();
+      if (win.fullScreen){
+        win.FullScreen.hideNavToolbox = win.oldHideNavToolbox;
+        win.FullScreen._updateToolbars = win.oldUpdateToolbars;
+      }
```
Comment (bwinton) [inline main.js]: We don't want to restore the functions if we start in fullscreen mode but don't end in fullscreen mode?  That seems a little odd…  (Or is it a typo where the if should check for `win.FullScreen`?)

File: verticaltabs.js [hunk 3]
```diff
@@ -126,6 +127,17 @@ VerticalTabs.prototype = {
         set('extensions.tabcentertest1@mozilla.com.lastUsedTimestamp', Date.now().toString());
         ss.setWindowValue(window, 'TCtoggledon', mainWindow.getAttribute('toggledon'));
         this.init();
+        if (mainWindow.getAttribute('F11-fullscreen') === 'true') {
+          let fullscreenctls = document.getElementById('window-controls');
+          let navbar = document.getElementById('nav-bar');
+          let toggler = document.getElementById('fullscr-toggler');
+          let sibling = document.getElementById('navigator-toolbox').nextSibling;
+          toggler.removeAttribute('hidden');
+          //hidden nav toolbox needs to be moved 1 pix higher to account for the toggler every time it hides
```
Comment (bwinton) [inline verticaltabs.js]: We should remove this comment, cause we aren't adding or removing a pixel here anymore…

File: verticaltabs.js [hunk 4]
```diff
@@ -126,6 +127,17 @@ VerticalTabs.prototype = {
...
+          window.FullScreen._updateToolbars.bind(window.FullScreen)(true);
```
Comment (bwinton) [inline verticaltabs.js]: Hmm.  If we're calling this as `window.FullScreen._updateToolbars` we _shouldn't_ need to bind it to `window.FullScreen`…

File: verticaltabs.js [hunk 5]
```diff
@@ -778,6 +790,12 @@ VerticalTabs.prototype = {
       ss.setWindowValue(window, 'TCtoggledon', mainWindow.getAttribute('toggledon'));
       window.VerticalTabs.sendPing('tab_center_toggled_off', window);
       this.init();
+      if (mainWindow.getAttribute('F11-fullscreen') === 'true'){
+        window.FullScreen._updateToolbars.bind(window.FullScreen)(true);
```
Comment (bwinton) [inline verticaltabs.js]: (Ditto my comment about binding here.)

File: main.js (hunk 2 above)
Comment (ericawright) [inline main.js]: typo

**Pull request merged**